meteor/lib/api/triggers/actionFactory.ts

```diff
@@ -378,6 +378,17 @@ function createRundownPlaylistSoftActivateAction(
 	}
 }
 
+function createRundownPlaylistSoftDeactivateAction(): ExecutableAction {
```

Comment:
Could we declare functions after the first use of the function and not before? I have seen that we declare the functions before their use a lot of places in Sofie and I would like us to stop doing that since it make the file read "backwards" instead of linear.
Refer to "The Stepdown Rule" from Clean Code

Reply:
It might just be a habit. Some scripting languages does not allow you to call a function before it's declared.
I agree that it would be nice to clean up, but cleaning it up might also cause merge errors with NRK.

Reply:
So far I haven't experienced issues with placing them below. I can see that WebStorm defaults to place them above.